### PR TITLE
[WinUI] Don't dogfood net6 on WINUI

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change ###

WInUI needs versions installed as globally, so dogfood doesn't make us any good right now.

So we need to also make sure we have the latest public preview of dotnet isntalled globally

### Additions made ###

* Adds 

### PR Checklist ###

- [ ] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)
- [ ] Targets a single property for a single control (or intertwined few properties)
- [ ] Adds the property to the appropriate interface
- [ ] Avoids any changes not essential to the handler property
- [ ] Adds the mapping to the PropertyMapper in the handler
- [ ] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [ ] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [ ] Tags ported renderer methods with [PortHandler]
- [ ] Adds an example of the property to the sample project (MainPage)
- [ ] Adds the property to the stub class
- [ ] Implements basic property tests in DeviceTests

#### Does this PR touch anything that might affect accessibility?
- [ ] Does this PR introduce a new control? (If yes, add an example using SemanticProperties to the SemanticsPage)
- [ ] APIs that modify focusability?
- [ ] APIs that modify any text property on a control?
- [ ] Does this PR modify view nesting or view arrangement in anyway?
- [ ] Is there the smallest possibility that your PR will change accessibility? 
- [ ] I'm not sure, please help me

If any of the above checkboxes apply to your PR, then the PR will need to provide testing to demonstrate that accessibility still works.

### CODE REVIEW

**Comment by rmarinho** [comment]
@vs-mobiletools-engineering-service2 backport release/6.0.1xx-preview4

**Comment by jonathanpeppers** [comment]
What was the actual error you were getting before this change?

Shouldn't that be in the commit message/PR description?

File: build.cake [hunk 1]
```diff
@@ -1236,20 +1235,37 @@ T GetBuildVariable<T>(string key, T defaultValue)
...
-        StartProcess("powershell",
-            new ProcessSettings
-            {
-                Arguments = new ProcessArgumentBuilder()
-                    .Append("start")
-                    .Append(sln)
-            });
+        if(usePreviewVs)
+        {
+            var vsLatest = VSWhereLatest(new VSWhereLatestSettings { IncludePrerelease = true, });
```
Comment (jonathanpeppers) [inline build.cake:1247]: Eilon was doing this on the powershell side here: https://github.com/dotnet/maui/pull/806

Should we pick a side? powershell or cake?

**Comment by Eilon** [comment]
Is this a change that we want to take ASAP or is it blocked on something before we merge?

**Comment by rmarinho** [comment]
@PureWeen  do we need this ?

File: build.cake (hunk 1 above)
Comment (jonathanpeppers) [inline build.cake:1247]: Does this need to be:
```suggestion
            var vsLatest = VSWhereLatest(new VSWhereLatestSettings { IncludePrerelease = usePreviewVs });
```

**Comment by PureWeen** [comment]
Closing in favor of https://github.com/dotnet/maui/pull/926

**Pull request closed without merging**